.changeset/gentle-deers-stare.md

```diff
@@ -0,0 +1,6 @@
+---
+"@wso2is/console": minor
+"@wso2is/i18n": patch
```

Comment:
Bump myaccount and identity-apps-core

Reply:
Fixed with 466fe513fbca5ed9dc13f15d2633888895fb948b